test/fixtures/user_fixtures.js

```diff
@@ -179,27 +179,41 @@ const buildRoleWithDefaults = (opts = {}) => {
 }
 
 function buildUserWithDefaults (opts) {
-  const serviceRoles = opts.service_roles ? lodash.flatMap(opts.service_roles, buildServiceRole) : [buildServiceRole()]
-  const data = {
-    external_id: opts.external_id || '7d19aff33f8948deb97ed16b2912dcd3',
-    username: opts.username || 'some-user@gov.uk',
-    email: opts.email || 'some-user@gov.uk',
-    otp_key: opts.otp_key || 'krb6fcianbdjkt01ecvi08jcln',
-    telephone_number: opts.telephone_number || '9127979',
-    service_roles: serviceRoles,
-    second_factor: opts.second_factor || 'SMS',
-    provisional_otp_key: opts.provisional_otp_key || 'a-provisional-key',
-    provisional_otp_key_created_at: opts.provisional_otp_key_created_at || null,
-    disabled: opts.disabled || false,
-    login_counter: opts.login_counter || 0,
-    session_version: opts.session_version || 0,
-    _links: opts._links || [{
+  lodash.defaults(opts, {
+    external_id: '7d19aff33f8948deb97ed16b2912dcd3',
+    username: 'some-user@gov.uk',
```

Comment:
I know it was `@gov.uk` before, but could you change it to `@example.com` or some invalid domain name. We try to not use "real" emails if we see ones in the old code. Thank you :)